app/assets/stylesheets/govuk_publishing_components/components/_modal-dialogue.scss

```diff
@@ -136,7 +138,10 @@ $govuk-modal-wide-breakpoint: $govuk-page-width + $govuk-modal-margin * 2 + $gov
 
   &:focus,
   &:hover {
+    @include govuk-focused-text;
+
     outline: none;
+    box-shadow: none;
```

Comment:
Probably outside of scope, but I think the close button looks pretty good with the box shadow. Certainly helps with the contrast between its focussed state and the white space below.

With:

<img width="741" alt="Screen Shot 2019-08-05 at 17 23 17" src="https://user-images.githubusercontent.com/861310/62479617-c9860080-b7a5-11e9-85f6-1513fe4f8356.png">

Without:

<img width="743" alt="Screen Shot 2019-08-05 at 17 23 09" src="https://user-images.githubusercontent.com/861310/62479603-becb6b80-b7a5-11e9-8796-f46500551906.png">

Reply:
Good call for potential improvement

Reply:
Added a commit to improve the focus on the modal's close button
<img width="1039" alt="Screen Shot 2019-08-07 at 11 11 17" src="https://user-images.githubusercontent.com/788096/62614871-25ff3200-b904-11e9-8258-7d0b4f604d55.png">